### PR TITLE
Vendor in latest containers/storage and containers/image

### PR DESCRIPTION
**Pull request by rhatdan**
Container image also switched out the version of bolddb
we were uisng.

Signed-off-by: Daniel J Walsh <dwalsh@redhat.com>

### CODE REVIEW

**Comment by rhatdan** [comment]
@nalind @vrothberg @TomSweeneyRedHat @giuseppe PTAL

File: util/util.go [hunk 1]
```diff
@@ -112,7 +112,7 @@ func ResolveName(name string, firstRegistry string, sc *types.SystemContext, sto
 	}
 	for _, registry := range searchRegistries {
 		if !registry.Blocked {
-			registries = append(registries, registry.URL)
+			registries = append(registries, registry.Location)
 		}
 	}
 	searchRegistriesAreEmpty := len(registries) == 0
```
Comment (TomSweeneyRedHat) [inline util/util.go]: DId you mean for these four files to be changed?  I'd really rather just have the vendor bump for image/storage to be in this PR, then the new bolt stuff in a separate one.

Comment (rhatdan) [reply util/util.go]: These changes were required by the new version of containers/image.

Comment (TomSweeneyRedHat) [reply util/util.go]: Okey Doke, thanks for the ptr.

**Comment by TomSweeneyRedHat** [comment]
LGTM

**Comment by rhatdan** [comment]
@nalind @vrothberg @pixdrift  @giuseppe PTAL

**Comment by vrothberg** [comment]
@rh-atomic-bot r+

**Comment by vrothberg** [comment]
Network flake. Restarted the job.

**Pull request closed without merging**